src/e2e/java/teammates/e2e/util/LNPSpecification.java

```diff
@@ -60,4 +54,37 @@ private void checkErrorLimit(double errorPct) {
                     + "% higher than the specified threshold. ";
         }
     }
+
+    /**
+     * Returns a builder for {@link LNPSpecification}.
+     */
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    /**
+     * A builder class for {@link LNPSpecification}.
+     */
+    public static class Builder {
```

Comment:
I think the rule suppression should be specific to the file rather than to the whole package. Take a look at `ResourceEndpoints` to see how to do this.

Reply:
updated the suppression to file-level

Reply:
I think using `//CHECKSTYLE.OFF:MissingJavadocMethod` (and later turn it on `//CHECKSTYLE.ON:MissingJavadocMethod`) is better than put it in the `suppression` file.

Reply:
Agree. But I find this will result inconsistency in suppression style with other existing Builder class. (specifically in `teammates/common/datatransfer/`). 

I feel it might be better to have another PR to improve suppression style for Builder across board?

Reply:
Please note that the suppression for those `datatransfer` has `<!-- To be implemented slowly -->` comment. It is not a convention. It is just we do not have time to add those Javadoc :P

Reply:
Alright! Updated.